File: rasterizer-dotnet-vb/GifWithWebPalette.vb
Imports ceTe.DynamicPDF.Rasterizer

Public Class GifWithWebPalette
        Friend Shared Sub Run()
            Dim rasterizer As New PdfRasterizer(Program.GetResourcePath("DocumentA.pdf"))
            Dim gifImageFormat As New GifImageFormat()
            gifImageFormat.DitheringAlgorithm = DitheringAlgorithm.FloydSteinberg
            gifImageFormat.DitheringPercent = 100
            gifImageFormat.ColorPalette = Palette.WebPalette
            gifImageFormat.AntiAliasing = True
            Dim fixedImageSize As New FixedImageSize(595, 841)
            rasterizer.Draw(Program.GetOutputDocPath("gif-web-palette-output.gif"), gifImageFormat, fixedImageSize)
        End Sub
    End Class

File: rasterizer-dotnet-examples-csharp/GifWithWebPalette.cs
using ceTe.DynamicPDF.Rasterizer;

namespace rasterizer_dotnet_csharp
{
    public class GifWithWebPalette
    {
        internal static void Run()
        {
            PdfRasterizer rasterizer = new PdfRasterizer(Program.GetResourcePath("DocumentA.pdf"));
            GifImageFormat GifImageFormat = new GifImageFormat();
            GifImageFormat.DitheringAlgorithm = DitheringAlgorithm.FloydSteinberg;
            GifImageFormat.DitheringPercent = 100;
            GifImageFormat.ColorPalette = Palette.WebPalette;
            GifImageFormat.AntiAliasing = true;
            FixedImageSize fixedImageSize = new FixedImageSize(595, 841);
            rasterizer.Draw(Program.GetOutputDocPath("gif-web-palette-output.gif"), GifImageFormat, fixedImageSize);
        }
    }
}